helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -198,6 +198,14 @@ processors:
         value: "{{ .Values.environment }}"
   {{- end }}
 
+  {{- if .Values.isWindows }}
+  metricstransform:
+    transforms:
+      - include: container.memory.working_set
+        action: insert
+        new_name: container_memory_usage_bytes
```

Comment:
I believe this metrics represented as `container.memory.usage` in otel conventions, and will be translated on the backend to `container_memory_usage_bytes` if needed. please validate usage of `container.memory.usage` instead.

Reply:
It also works with `container.memory.usage`, so I have made this change.